src/Microsoft.AspNetCore.Mvc.Core/ApiExplorer/IErrorPolicy.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    public interface IErrorPolicy
```

Comment:
Unsure currently. I would rather see the O/M for problem fleshed out first rather than focus on the deets of how the conventions for errors work.